content/index.pl.md

```diff
@@ -0,0 +1,163 @@
+
+Kryteria sukcesu określają „zgodność” z WCAG. Oznacza to, że aby spełnić wymagania WCAG, treść musi spełniać kryteria sukcesu. Szczegóły znajdują się w [WCAG w sekcji Zgodność](https://www.w3.org/TR/WCAG22/#conformance).
+
+Krótkie podsumowanie wytycznych WCAG 2 można znaleźć w artykule **[[WCAG 2 w skrócie]](/standards-guidelines/wcag/glance/)**.
+		
+### Materiały pomocnicze i dodatkowe wskazówki {#supplement}
+
+Poza WCAG, poniższe materiały pomagają zrozumieć i wdrażać WCAG oraz poprawiać dostępność:
+* Krótki przewodnik / Jak spełnić WCAG 2 / Lista kontrolna
+* Objaśnienia WCAG 2
+* Techniki dla WCAG 2
+* Zasady testowania dla WCAG 2
+* Dodatkowe wytyczne
+
+**Zapoznaj się z tymi zasobami WCAG 2 w [dokumentacji WCAG 2](/standards-guidelines/wcag/docs/).**
+
+## WCAG 2.0, 2.1, 2.2 {#versions}
+
+Standardy Wytyczne dla dostępności treści internetowych (WCAG) są stabilne i można się do nich odwoływać,  kiedy są opublikowane jako 'Rekomendacja W3C'.
+
+* [WCAG 2.0](https://www.w3.org/TR/WCAG20/) zostały opublikowane 11 grudnia 2008 roku.
+* [WCAG 2.1](https://www.w3.org/TR/WCAG21/) zostały opublikowane 5 czerwca 2018 r., a ich aktualizacja została opublikowana 21 września 2023 r. Artykuł [WCAG 2 FAQ](https://www.w3.org/WAI/standards-guidelines/wcag/faq/) zawiera informacje na temat aktualizacji.
+* [WCAG 2.2](https://www.w3.org/TR/WCAG22/) zostały opublikowane 5 października 2023 roku.
+
+WCAG 2.0, 2.1 i 2.2 zostały zaprojektowane tak, aby były „kompatybilne wstecznie”, co oznacza, że treści zgodne z WCAG 2.2 są również zgodne z WCAG 2.1 i WCAG 2.0. Jeśli chcesz zachować zgodność ze wszystkimi wersjami, możesz użyć zasobów WCAG 2.2 i nie musisz się martwić o sprawdzanie poprzednich wersji.
+
+Wszystkie kryteria sukcesu z 2.0 są zawarte w 2.1, a wszystkie z 2.1 są zawarte w 2.2 (z wyjątkiem 4.1.1, jak wyjaśniono w następnym akapicie).
+
+* WCAG 2.0 mają 12 wytycznych.
+* WCAG 2.1 dodają 1 wytyczną i 17 kryteriów sukcesu. Są one omówione w dokumencie [Nowe w WCAG 2.1](/standards-guidelines/wcag/new-in-21/).
+* WCAG 2.2 dodają 9 kryteriów sukcesu. Są one omówione w dokumencie [Nowe w WCAG 2.2](/standards-guidelines/wcag/new-in-22/).
+		
+Kilka rzeczy uległo zmianie, a aktualizacje w odpowiednich dokumentach mają zapewnić kompatybilność wsteczną w praktyce. Główną zmianą jest to, że w WCAG 2.2 jedno kryterium sukcesu (4.1.1 Poprawność kodu) jest przestarzałe. Uwagi dodane do errat WCAG 2.1 i WCAG 2.0 odnoszą się do tego, jak wyjaśniono w [FAQ dla WCAG 2, 4.1.1 Poprawność kodu](/standards-guidelines/wcag/faq/#parsing411). WCAG 2.2 zawiera również uwagi dotyczące różnych języków; więcej informacji można znaleźć w [FAQ dla WCAG 2, internationalizacja](/standards-guidelines/wcag/faq/#i18n22).
+
+WCAG 2.0, WCAG 2.1 i WCAG 2.2  to obowiązujące standardy.  WCAG 2.2 nie deprecjonuje ani nie zastępuje WCAG 2.1, a WCAG 2.1 nie deprecjonuje ani nie zastępuje WCAG 2.0. W3C zachęca do korzystania z najnowszej wersji WCAG.
```

Comment:
Original text:
> WCAG 2.0, WCAG 2.1, and WCAG 2.2 are all existing standards.

Polish translation (automatically re-translated into English):
> WCAG 2.0, WCAG 2.1, and WCAG 2.2 are the applicable standards.

It seems to me that we are losing the idea that "all" standards are currently applicable, as described in the following sentences.